### PR TITLE
Fixes #8711: Improve 4.0 menu

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/8711

### CODE REVIEW

**Comment by RaphaelGauthier** [comment]
PR updated

**Comment by RaphaelGauthier** [comment]
PR updated

File: rudder-core/pom.xml [hunk 1]
```diff
@@ -77,7 +77,11 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
       <artifactId>commons-io</artifactId>
       <version>${commons-io-version}</version>
     </dependency>
-
+    <dependency>
+      <groupId>ca.mrvisser</groupId>
```
Comment (VinceMacBuche) [inline rudder-core/pom.xml]: Can you remove those lines from pom.xml ?

File: rudder-web/src/main/scala/com/normation/rudder/web/comet/AsyncDeployment.scala [hunk 2]
```diff
@@ -97,7 +99,7 @@ class AsyncDeployment extends CometActor with CometListener with Loggable {
 
   override def render = {
     partialUpdate(updateDuration)
-    new RenderOut(layout)
+    new RenderOut(layout)
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/comet/AsyncDeployment.scala]: can you remove the trailling space (at the end of the line)

**Comment by RaphaelGauthier** [comment]
PR updated

**Pull request merged**